scene/main/node.cpp

```diff
@@ -235,7 +235,7 @@ void Node::_propagate_enter_tree() {
 
 void Node::_propagate_exit_tree() {
 
-//block while removing children
+	//block while removing children
```

Comment:
This causes clang-format error.

Reply:
That's so weird...cause I ran ClangFormat in my VS2017 project and it made that change without me even realizing it. I guess it's broken or something?

Reply:
That's because you might have a different version of clang-format which behaves slightly differently. The CI uses clang-format 3.9.1.

Reply:
Is there a file for that clang-format that I can direct VS to in order to get it to follow those standards? Or would it need to be something like downloading another executable for clang-format 3.9.1 and then telling the VS extension to point to that or something? (don't know if you use VS at all) @akien-mga